website/content/api-docs/system/managed-keys.mdx

```diff
@@ -0,0 +1,194 @@
+
+- `allow_replace_key` `(string: "false")` - Controls the ability for Vault to replace through generation or importing
+  a key into the configured backend even if a key is present, if set to false those operations are forbidden
+  if a key exists.
+
+- `allow_store_key` `(string: "false")` - Controls the ability for Vault to generate or import a key within the
+  configured backend, if "false" those operations will be forbidden.
+
+- `any_mount` `(string: "false")` - Allow usage from any mount point within the namespace if "true". If "false"
+  specific mount points will need to be tuned setting `allowed_managed_keys` with the name value of the key
+  granting usage.
+
+#### PKCS#11 backend Parameters
+- `type` `(string: "pkcs11")` - To select a PKCS#11 backend, the type parameter must be set to `pkcs11`
+
+- `library` `(string: <required>)` - The name of the `kms_library` stanza to use from Vault's config to
+   lookup the local library path. See [kms_library stanza](/docs/configuration/kms-library) for further details.
+
+- `key_label` `(string: <required>)`: The label of the key to use. If the key
+  does not exist and generation is enabled, this is the label that will be given
+  to the generated key.
+
+- `mechanism` `(string: <required>)` - The encryption/decryption mechanism to use,
+  specified as a hexadecimal (prefixed by 0x) string. The following are supported mechanisms
+    - `0x1041` `CKM_ECDSA`
+    - `0x000D` `CKM_RSA_PKCS_PSS`
+    - `0x0009` `CKM_RSA_PKCS_OAEP`
+    - `0x0001` `CKM_RSA_PKCS`
+
+- `pin` `(string: <required>)`: The PIN for login.
+
+- `slot` `(string: <slot or token label required>)`: The slot number to use,
+  specified as a string (e.g. `"2305843009213693953"`).
+
+- `token_label` `(string: <slot or token label required>)`: The slot token label to
+  use.
+
+- `curve` `(string: "")` - supplies the curve value when using the `CKM_ECDSA`
+  mechanism. This key is required if  The supported values are
+    - `P384`
+    - `P521`
+    - `P256`
+
+- `key_bits` `(int: "")` - supplies the key bits when using `CKM_RSA_PKCS_PSS`,
+  `CKM_RSA_PKCS_OAEP` or `CKM_RSA_PKCS` mechanisms, supported values are
+    - 2048
+    - 3072
+    - 4096
+
+- `force_rw_session` `(string: "false")`: Force all operations to open up
+  a read-write session to the HSM. This is a boolean expressed as a string (e.g.
+  `"true"`). This key is mainly to work around a limitation within AWS's CloudHSM v5
+  pkcs11 implementation.
+
+## Read a Managed Key's properties
+
+This endpoint returns the configuration of the managed key at the given path.
+
+| Method | Path                            |
+|:-------|:--------------------------------|
+| `GET`  | `/sys/managed-keys/:type/:name` |
+
+### Parameters
+- `name` `(string: <required>)` - A unique lowercase name that serves as identifying the key. Must be
+  unique throughout the namespace.
+
+- `type` `(string: <required>)` – The backend type that will be leveraged for the managed key.
```

Comment:
will be leveraged implies it isn't yet.  Copy/paste from the create op?

Reply:
Yup addressed.